vllm/compilation/decorators.py

```diff
@@ -104,6 +105,7 @@ def support_torch_compile(
     dynamic_arg_dims: dict[str, int | list[int]] | None = None,
     mark_unbacked_dims: dict[str, int | list[int]] | None = None,
     enable_if: Callable[[VllmConfig], bool] | None = None,
+    shape_invariants: Callable[..., None] = lambda *args, **kwargs: None,
```

Comment:
Seems like shape_invariants gets compiled as part of forward, can you add this to method documentation? Also that wasn't clear in the doc as well.

Reply:
done